keyboards/kj_modify/rs40/info.json

```diff
@@ -0,0 +1,77 @@
+{
+    "manufacturer": "KJ-Modify",
+    "keyboard_name": "kj_modify/rs40",
+    "maintainer": "Audite Marlow",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B12", "B14", "B15", "A8", "A9", "C13", "B9", "B8", "B5", "B4", "B3", "A15"],
+        "rows": ["B13", "A10", "B1", "B10"]
+    },
+    "processor": "STM32F401",
+    "url": "",
```

Comment:
I think this has to link to some source where to buy/build the keyboard, perhaps the one at `README/Hardware Availability`